sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -289,8 +271,16 @@ export class ManagementClient extends LinkEntity {
     }
   }
 
+  protected createRheaLink(options: RequestResponseLinkOptions): Promise<RequestResponseLink> {
+    return RequestResponseLink.create(
+      this._context.connection,
+      options.senderOptions,
+      options.receiverOptions
+    );
+  }
+
   private _isMgmtRequestResponseLinkOpen(): boolean {
-    return this._mgmtReqResLink! && this._mgmtReqResLink!.isOpen();
+    return this.link! && this.link!.isOpen();
```

Comment:
Since `initLink()` will check if the link is open before trying to initialize one, do we still need `_isMgmtRequestResponseLinkOpen()`?

Reply:
There are a few nice changes we can make here but I'm going to reserve it for the next PR since we should add some tests for it.

1. Removing isOpen() is a solid change.
2. The abortSignal support is pretty-close to being fully redundant - we can let linkEntity's init(abortSignal) support take over and we have a pattern already for the "timeout operation" in our utils as well.
  (this would fix https://github.com/Azure/azure-sdk-for-js/issues/4422 that's mentioned in the code comments, but only for service bus, not event hubs)
3. There's also a lock taken around the init(). That code could migrate into LinkEntity as well which will help solve another issue (multiple simultaneous init's don't properly wait for each other). 

So I think I'll sweep up and get mgmt link in that one.

Reply:
Simplest change was just to change all calls ti isMgmtRequestResponseLinkOpen to calls to .isOpen() which does the same thing.

I filed an issue where I mention a few spots where I think we can easily remove the call but we just need to test it: #10656